### DISCUSSION TITLE
The Latest Clock Page not working

### DISCUSSION THREAD

**Discussion by AFireInside777** [codewars/codewars.com#3180] [Q&A] [answered]
I am trying to test my solutions for this kata on this page: https://www.codewars.com/kata/58925dcb71f43f30cd00005f/train/javascript
However, for some reason, I am now getting this Undefined error when I was not getting this before. My code works in VS Code, but not on Codewars.

![image](https://github.com/codewars/codewars.com/assets/49132327/621d8362-3412-405f-b28e-c093328968b3)

When I do test using Node V 8, it shows that the program is working but then shows this error: 
![image](https://github.com/codewars/codewars.com/assets/49132327/f8db99d4-4c76-4780-9f34-e49bb29f7183)

**Comment by AFireInside777** [comment, marked as answer]
Never mind, it's my program.

**Comment by Blind4Basics** [comment]
Hello,

Note that github discussions are supposed to be more generic. For problems with a kata, please post a `Question` in the discourse of the kata itself.

Cheers
